lldb/source/Plugins/LanguageRuntime/CPlusPlus/CPPLanguageRuntime.h

```diff
@@ -81,6 +81,8 @@ class CPPLanguageRuntime : public LanguageRuntime {
 
   bool IsSymbolARuntimeThunk(const Symbol &symbol) override;
 
+  static bool ShouldUseMicrosoftABI(Process *process);
```

Comment:
Rather than returning a bool, can we have an enum for the C++ runtime ABIs and have this return either Itanium or Microsoft?